### PR TITLE
Add support to new Typeform Responses Api

### PR DESCRIPTION
**Pull request by monsha**
Notes: The class Typeform::Form doesn't have major changes for backward compatibility, use Typeform::Response instead of. 

https://trello.com/c/Jr5mW7Z5/465-implement-a-new-typeform-huginn-agent

### CODE REVIEW

File: test/test_helper.rb [hunk 1]
```diff
@@ -4,7 +4,9 @@
 require 'vcr'
 
 VCR.configure do |c|
-  c.filter_sensitive_data("<TYPEFORM_API_KEY>") { Typeform::Typeform.api_key }
   c.cassette_library_dir = "test/fixtures"
   c.hook_into :webmock
+  c.before_record do |i|
+    i.response.body.force_encoding('UTF-8')
```
Comment (OpakAlex) [inline test/test_helper.rb:10]: can we use here `try`? `i.response.body.try(: force_encoding, 'UTF-8')`

**Review by OpakAlex — APPROVED** [review]

File: test/test_helper.rb (hunk 1 above)
Comment (duboff) [reply test/test_helper.rb:10]: Or `&.`?

Comment (OpakAlex) [reply test/test_helper.rb:10]: yes. But it's just helper. maybe we can ignore this.

Comment (monsha) [reply test/test_helper.rb:10]: I think is no necessary, HTTP responses always have a body

Comment (OpakAlex) [reply test/test_helper.rb:10]: ok

**Comment by monsha** [comment]
Close this PR because this branch is being used directly on Huginn repo.

**Pull request closed without merging**